### PR TITLE
Aligning the Dockerfile file to the Node.js app Docker guidelines

### PR DESCRIPTION
**Pull request by gtorodelvalle**
- No linting issues.
- 100% tests passed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
-RUN git clone https://github.com/telefonicaid/fiware-sth-comet.git
+MAINTAINER Germán Toro del Valle <german.torodelvalle@telefonica.com>
 
-WORKDIR  /github/telefonicaid/fiware-sth-comet
-RUN git fetch && git checkout master && npm install
+RUN yum update -y && yum install -y wget \
+  && wget http://ftp.rediris.es/mirror/fedora-epel/6/i386/epel-release-6-8.noarch.rpm && yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm \
+  && yum install -y npm git
 
-EXPOSE 8666
```
Comment (dmoranj) [inline Dockerfile]: Why did you remove the "EXPOSE" command?

**Comment by AlvaroVega** [comment]
LGTM
